server/src/main/java/org/elasticsearch/TransportVersions.java

```diff
@@ -211,6 +211,7 @@ static TransportVersion def(int id) {
     public static final TransportVersion ML_INFERENCE_AMAZON_BEDROCK_ADDED = def(8_702_00_0);
     public static final TransportVersion ENTERPRISE_GEOIP_DOWNLOADER_BACKPORT_8_15 = def(8_702_00_1);
     public static final TransportVersion FIX_VECTOR_SIMILARITY_INNER_HITS_BACKPORT_8_15 = def(8_702_00_2);
+    public static final TransportVersion ESQL_ATTRIBUTE_CACHED_SERIALIZATION_8_15 = def(8_702_00_3);
```

Comment:
Nit: maybe some javadoc to say that this is the squashed transport version changes for both caching mechanisms?